packages/story-editor/src/components/canvas/pageAttachment/index.js

```diff
@@ -144,7 +144,10 @@ function PageAttachment({ pageAttachment = {} }) {
     icon,
     theme,
   } = pageAttachment;
-
+  const {
+    isRTL,
+    styleConstants: { topOffset },
```

Comment:
If I understand correctly, this is a value that could potentially come from the backend (but is not set now)? So currently, it's `0` (the default) and would still need defining `32` in the backend?

Wondering if we could make the name more explanatory, e.g. `styleConstants: { editorTopOffset },` or something similar? it's currently not clear what kind of top offset is meant.

An alternative would be to actually calculate the real top offset once the editor has loaded instead of taking a static value from the backend (e.g. we're calculating the real page size once it has loaded as well), thoughts? 

(In case I understand correctly that's what this `topOffset` means here)

Reply:
Yes the `topOffset` value comes from the `wp-story-editor` where it has been defined as a constant so whoever wants to integrate may pass it the same way we are doing. I am fine doing it either way but if it were to be calculated once page loads then i guess it would require another PR

Reply:
Ah, okay, thought that this was supposed to come from the backend but coming from `wp-story-editor` makes sense.

Reply:
@miina I agree, `styleConstants: { editorTopOffset }` would be more explanatory 👍  . We may do it now or I will raise a PR for it. 
Calculating the real top offset looks like a natural choice here however we also have a collapse sidebar menu involved, so we would need some observer/listener for it. See the discussion in https://github.com/google/web-stories-wp/pull/10119